server/service/src/stock_line/tests/query.rs

```diff
@@ -22,14 +22,14 @@ mod query {
             service.get_stock_lines(
                 &context,
                 Some(PaginationOption {
-                    limit: Some(2000),
+                    limit: Some(6000),
```

Comment:
Side quest, this test was broken in main...

Reply:
thanks